### ISSUE TITLE
Use existing CAD model and embed it on the website.

### ISSUE THREAD

**Issue by stefek99** [astralship/astralship.github.io#18]
New model is being developed but it can take a while - why not use the existing one?

EDIT: we could use some of the tools to embed on the website: http://babylonjs.com/

**Comment by quantumLeaf** [comment]
CAD model wouldn't be directly hostable. There's potentially a discussion to be had with Dave whether he wants to share his work publically while still in progress. If he does fine, if not also fine.

**Comment by stefek99** [comment]
Not disputing or anything, you can easily embed 3D objects on the website: http://babylonjs.com

**Comment by quantumLeaf** [comment]
Yes, we embed graphics, maybe there is also a player for fusion 360
objects. But I don't think it's worth wasting time on. Dave can export a
graphics object from his software and we can render it with webgl.

However, I think we already have plenty of stuff out there. If we don't get
any interest with this, a cad model isn't going to make much difference.

**Comment by stefek99** [comment]
@richhop Do you mind sharing a current progress?

To showcase that we have something is already a step forward...

People could get excited seeing a VR 3D model...

Everything we do is "work in progress" so don't worry if it is not finished :)

**Comment by stefek99** [comment]
![astralship-floor](https://cloud.githubusercontent.com/assets/249703/25302798/710d8814-273e-11e7-9ef7-3161ef0d4af7.gif)

**Comment by stefek99** [comment]
Related to #62 

We are technology guys... Let's showcase our ability!
